governance-and-automation/security-compliance-checks/tenant-storage.md

```diff
@@ -9,9 +9,9 @@ Syskit Point mitigates the risk of running out of storage based on the settings
 
 The purpose of the Tenant Storage Limit check is to **notify you when your storage percentage gets close to the limit you set**. This information can help you determine whether you should change the storage limit set for the policy, remove any redundant content to free up space or purchase additional storage from Microsoft.
 
-The Tenant Storage policy is **a tenant-wide policy, and it cannot be automated**. That means that no tasks are created to resolve this policy violation, and no emails are sent to collaborators.
+The Tenant Storage policy is **a tenant-wide policy, and it can be automated**. When automation is enabled, Syskit Point creates tasks and sends emails to users as defined in the policy settings. To learn more, take a look at the [Tenant Storage Limit policy article](../../governance-and-automation/automated-workflows/tenant-storage-admin.md).
 
-When a tenant storage limit is exceeded, Syskit Point **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails. 
+By default, the automation is turned on for this policy. If the policy automation is turned off, when the tenant storage limit is exceeded, Syskit Point still **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but if the policy automation is turned off, it does not create tasks or send any emails to workspace owners. 
```

Comment:
send any emails to Syskit Point Administrators. ovdje se radi o adminima više nego ownerima